### DISCUSSION TITLE
小白构建失败求助

### DISCUSSION THREAD

**Discussion by Yusakie** [imsyy/home#314] [Q&A]
`yarn run v1.22.22
$ vite build
vite v4.5.3 building for production...
(!) %VITE_SITE_LOGO% is not defined in env variables found in /index.html. Is the variable mistyped?
(!) %VITE_SITE_APPLE_LOGO% is not defined in env variables found in /index.html. Is the variable mistyped?
(!) %VITE_SITE_APPLE_LOGO% is not defined in env variables found in /index.html. Is the variable mistyped? (x2)
(!) %VITE_SITE_APPLE_LOGO% is not defined in env variables found in /index.html. Is the variable mistyped? (x3)
(!) %VITE_SITE_DES% is not defined in env variables found in /index.html. Is the variable mistyped?
(!) %VITE_SITE_KEYWORDS% is not defined in env variables found in /index.html. Is the variable mistyped?
(!) %VITE_SITE_ANTHOR% is not defined in env variables found in /index.html. Is the variable mistyped?
(!) %VITE_SITE_NAME% is not defined in env variables found in /index.html. Is the variable mistyped?
✓ 0 modules transformed.
[vite-plugin-pwa:build] URI malformed
file: /www/wwwroot/home/index.html
error during build:
URIError: URI malformed
    at decodeURI (<anonymous>)
    at file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37635:45
    at traverseNodes (file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37435:5)
    at file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37439:48
    at Array.forEach (<anonymous>)
    at traverseNodes (file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37439:25)
    at file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37439:48
    at Array.forEach (<anonymous>)
    at traverseNodes (file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37439:25)
    at file:///www/wwwroot/home/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:37439:48
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`
![image](https://github.com/imsyy/home/assets/87126204/ddffd65d-40a9-4d69-9121-8e53c3d400ca)
![image](https://github.com/imsyy/home/assets/87126204/569d0cce-3665-4895-ba65-02809d8216b3)

**Comment by WhyLIM** [comment]
你是不是没有复制一份 `.env` 文件，`.env.example` 只是一个示例，真正引用的是 `.env`

**Reply by changaxj** [reply]
问一下 复制哪里的文件呀？？

**Reply by WhyLIM** [reply]
复制 `/.env.example` 文件并重命名为 `/.env`，然后直接修改 `/.env` 的内容

**Reply by changaxj** [reply]
非常感谢！！ 然后 这个和我当初那一版本 好像没有了音乐播放器  我能自己再加上吗？有没有解决办法 (作者的哪个音乐播放器)

**Reply by WhyLIM** [reply]
不好意思，忘记回复了。。不太明白你的情况，使用音乐播放器需要自己部署meting api，教程在 `/.env.example` 文件中亦有提及
